Exercise_1.java

```diff
@@ -1,35 +1,65 @@
-class Stack { 
-    //Please read sample.java file before starting.
-  //Kindly include Time and Space complexity at top of each file
+// Time Complexity : O(1)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : No
+class Stack{
     static final int MAX = 1000; 
     int top; 
     int a[] = new int[MAX]; // Maximum size of Stack 
   
     boolean isEmpty() 
     { 
         //Write your code here 
+        if(top < 0)

```

Comment:
If top < 0 i.e. if top == -1 then it that case you need to return true (stack is empty)